fb_apcupsd/recipes/default.rb

```diff
@@ -0,0 +1,42 @@
+#
+# Cookbook Name:: fb_apcupsd
+# Recipe:: default
+#
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+#
+# Copyright (c) 2016-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+
+unless node.debian?
+  fail 'fb_apcupsd is only supported on Debian.'
+end
+
+package 'apcupsd' do
+  action :upgrade
+end
+
+# this isn't a template as there's nothing really useful to configure in it
+cookbook_file '/etc/default/apcupsd' do
```

Comment:
wanna spin up a redhatish vm and see if you can make this work as a sysconfig file as well?